cmd/da-light-client/start/start.go

```diff
@@ -53,6 +60,7 @@ func Cmd() *cobra.Command {
 
 func addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP(rpcEndpointFlag, "", celestia.DefaultCelestiaRPC, "The DA rpc endpoint to connect to.")
+	cmd.Flags().StringP(metricsEndpointFlag, "", "", "The OTEL collector metrics endpoint to connect to.")
```

Comment:
what is OTEL? isn't it prometheus?

Reply:
No - celestia light node push metrics to on OTEL collector instead of publishing a prometheus HTTP metric server, can be used for preprocessing and filtering